### ISSUE TITLE
Reordering a structure does not change the uri on globe icon

### ISSUE THREAD

**Issue by bartrylant** [craftcms/cms#2692]
### Description
When dragging structure items around (making children or children's children) the uri's from the globe icon don't change.

### Steps to reproduce
1. Drag an item in a structure to the right so it becomes a child of the entry above
2. Click the globe icon on the right
3. Still goes to the old uri, not the new uri

### Additional info
- Craft version: Craft Pro 3.0.1
- PHP version: 7.1.16
- Database driver & version: MySQL 5.7.21

**Comment by brandonkelly** [comment]
Yeah this is a known issue. The problem is that the URIs are updated via a queue job, rather than immediately after the reorder is complete. So unless we start polling for changes (which would be expensive), it's not really possible for us to fix it.

I'll leave the issue open, though, as it's a legit bug.

**Comment by brandonkelly** [comment]
@Tyrannogina Pretty hard to know where to start without that. Can you send us a ticket from your Craft Support widget on your Dashboard?
